modules/social_features/social_private_message/src/Plugin/UserExportPlugin/UserAnalyticsPrivateMessage.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Drupal\social_private_message\Plugin\UserExportPlugin;
+
+use Drupal\social_user_export\Plugin\UserExportPluginBase;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\user\UserInterface;
+
+/**
+ * Provides a 'UserAnalyticsPrivateMessage' user export row.
+ *
+ * @UserExportPlugin(
+ *  id = "user_private_message",
+ *  label = @Translation("Number of Private messages"),
+ *  weight = -190,
+ * )
+ */
+class UserAnalyticsPrivateMessage extends UserExportPluginBase {
+
+  use StringTranslationTrait;
```

Comment:
@jaapjan do we need this trait in every file? Even if the `UserExportPluginBase` already uses it?

Reply:
Nope, good point. I'll remove it.